README.md

```
@@ -0,0 +1,92 @@
+├── tests/                # Vitest ユニット / 契約テスト
+├── e2e/                  # Playwright E2E テスト
+└── CLAUDE.md             # AI エージェント / 開発規約
+```
+
+---
+
+## 開発コマンド
+
+パッケージマネージャには **Bun** を推奨しています。
+
+### セットアップ
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)